css/style.css

```diff
@@ -0,0 +1,84 @@
+body {
+    background-color: whitesmoke;
```

Comment:
the use of the CSS box model will be very helpful while trying to apply some styling properties